__tests__/components/TransactionList/TransactionListFilters/TransactionListFilter.spec.ts

```diff
@@ -25,8 +38,21 @@ describe('TransactionListFilters', () => {
         vm.onSignerSelectorChange('TAD5BAHLOIXCRRB6GU2H72HPXMBBVAEUQRYPHBY');
         expect(vm.$store.dispatch).toBeCalledWith('account/SET_CURRENT_SIGNER', { address: addr, reset: true, unsubscribeWS: false });
     });
+
     test("should not call the 'account/SET_CURRENT_SIGNER' without address", () => {
         vm.onSignerSelectorChange();
         expect(vm.$store.dispatch).not.toBeCalled();
     });
+
+    test("should call 'transaction/filterTransactions' with blacklisted contacts", () => {
```

Comment:
would be good to add a test when `isBlackListFilterActivated == true` to cover else branch too